### ISSUE TITLE
Only 44 instead of 45 sequences

### ISSUE THREAD

**Issue by Starsonic007** [rfcx/rfcx-console#93]
In staging, I have found that there are only 44 instead of 45 individual sequences in the file with the GUID 4ccdbe0e-722a-4bd5-a19a-0188c4b92974

![image](https://cloud.githubusercontent.com/assets/20114180/18524189/af2aa034-7ab7-11e6-99b6-b726aede1f9a.png)

**Comment by rassokhin-s** [comment]
hey @Starsonic007 
First of all I guess you've mentioned wrong audio guid :) I see `80ca73d1-8ed9-4d07-b424-bf51fa9e092f` on the screenshot.
I've checked audio with guid `80ca73d1-8ed9-4d07-b424-bf51fa9e092f` and it has duration `89331ms`. We create a window for each **full** 2 seconds. In case of this file there is not enough seconds to create 45th window, that's why there is no window. If this file had duration >= 90000ms, then we would be able to create 45 windows for it.
We have already talked with @daebwae about this situation. Windows count depends exactly on audio duration.
